dashboard/app/controllers/api/v1/projects/public_gallery_controller.rb

```diff
@@ -5,11 +5,22 @@ def index
       expires_in 5.seconds, public: true # cache
     end
 
-    render json: ProjectsList.fetch_published_projects(
-      params[:project_type],
-      limit: params[:limit],
-      published_before: params[:published_before]
-    )
+    if params[:showFeatured] == "1"
+      render json: ProjectsList.fetch_published_projects(
```

Comment:
This render will eventually be swapped with `fetch_featured_published_projects` because those are the ones we want to show, but I'm not there yet.